lib/content-services/search/search-query-builder.service.ts

```diff
@@ -384,4 +386,12 @@ export class SearchQueryBuilderService {
 
         return null;
     }
+
+    getSupportedLabel(item: string) {
```

Comment:
missing return type, is the item the right name for this input parameter?

Reply:
changed to a more suggestive name